packages/cli/src/commands/server.ts

```diff
@@ -0,0 +1,34 @@
+import { Command, flags as oflags } from '@oclif/command';
+import { createServer } from '../util/createServer';
+
+export default class Server extends Command {
+  public static description = 'Start a server with the given spec file';
+  public static flags = {
+    port: oflags.integer({
+      char: 'p',
+      description: 'Port that Prism will run on.',
+      default: 4010,
```

Comment:
Yeah... I know it's an overkill to refactor something if there are only two repetitions but at the same time it just asks for a refactor in this case :p 

I'm not gonna push it though, I will leave it up to your sense of good taste ;)

Reply:
Duplication is far cheaper than the wrong abstraction. If this happens 3 times I'd consider to move it somewhere shared. For now it'll just do fine.

Reply:
Like I said, I don't care that much and I agree with @XVincentX . But in case we'll release a package that will expose `mock` on `4010` port and `forward` on port `4011` because we forgot to update both ports you've been warned ;)

Reply:
Let's bring that magic number somewhere else then? Just that value?

Reply:
:+1: good trade off ;)

Reply:
I agree with @XVincentX that we could wait with that until we actually face a problem with duplication. However extracting PORT only won't help much - we have descriptions and other "const" values. So I moved args and flags to shared place. Please comment if you like it :)

Reply:
I like the solution. Ok by me.

Reply:
I'm good!